pom.xml

```diff
@@ -1,37 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
```

Comment:
Please, revert changes to the POM file. We are not changing it unless totally necesary. You may keep it modified in your own fork, but we are keeping it as is in the main branch